prisma/seedDev.mjs

```diff
@@ -65,6 +69,26 @@ function randomTherapyPrices(selectedTherapies) {
   return therapyPrices;
 }
 
+function randomWorkTime() {
+  const weekdays = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN'];
+  let time;
+  let workTimeData;
+  let isDayOff;
```

Comment:
Why do you need these variables outside map-function? )

Reply:
Don't really need it there, changed it